catalogue/webapp/components/PaletteSimilarityBox/PaletteSimilarityBox.js

```diff
@@ -25,18 +22,18 @@ const PaletteSimilarityBox = ({ work }: Props) => {
   return similar ? (
     <div>
       <div className="flex">
-        {similar.neighbours.map(neighbour => {
+        {similar.map(image => {
           return (
             <div
               style={{
```

Comment:
Don't know if you've come across what we've called [`root-scope-classes`](https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/styles/utilities/_root-scope-classes.scss) – our own, not-always-consistentenly-named, utility classes. I think I'd tend to favour using one of those where it makes sense (I think we tend to be able to achieve most flexbox-y stuff with them). And probably a Space component for the margin-right.

Although I'm unsure whether we're considering any of this to be prototype, or if it's going to land in prod after we've had it behind a toggle for a bit. If it's prototype, then I'm much less fussy.

Reply:
Agreed - this code predates this PR by a while, I just updated the request bit to use the js service I'd written. This is behind a different toggle and I imagine will be removed at some point in the future...